Makefile

```diff
@@ -62,3 +70,15 @@ generate-code:
 patch: generate-code
 	awk '($$1 != "\"github.com/gorilla/mux\"") {print}' auto-generated/api/go/api_default.go > ${TMP_FILE}
 	mv ${TMP_FILE} auto-generated/api/go/api_default.go
+
+.PHONY: build
```

Comment:
this rule is called `source-build` in fybrik, can you rename it pls.

Reply:
replaced `compile` with `source-build`
and `build` with `docker-build`